substrate/bin/node/cli/src/cli.rs

```diff
@@ -61,11 +61,6 @@ pub enum Subcommand {
 	#[command(subcommand)]
 	Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 
-	/// Try-runtime has migrated to a standalone CLI
-	/// (<https://github.com/paritytech/try-runtime-cli>). The subcommand exists as a stub and
-	/// deprecation notice. It will be removed entirely some time after January 2024.
```

Comment:
damn we didn't mark this as deprecated before, so it will be a breaking chaneg in all node side code of all teams... wdyt about making it deprecated now, and removing one version later? we can still fully remove it from our codebase.

Reply:
The actual command has been a noop for a few releases, it only logged that it's deprecated with instructions to use the dedicated cli :)